client/my-sites/site-settings/podcast-cover-image-setting/style.scss

```diff
@@ -24,6 +23,9 @@
 	align-self: center;
 	background: $transparent;
 	position: relative;
+	max-height: 100%;
+	max-width: 100%;
+	margin: 0 auto;
```

Comment:
This is just extra to horizontally center images that are smaller than the preview container.